File: src/main/java/EspressoShot.java
public class EspressoShot extends ToppingDecorator{

    public EspressoShot(EspressoBeverage newEspBev) {
        super(newEspBev);
        System.out.println("Adding Espresso Shot");
        System.out.println("Adding Steamed Milk");
    }
    public String getDescription(){
        return tempEspresso.getDescription() + ", Extra Espresso Shot";
    }
    public double getCost(){
        return tempEspresso.getCost() + 1.00;
    }

}

File: Demohahaha/EspressoShot.cs
class EspressoShot : ToppingDecorator{

    EspressoShot(EspressoBeverage newEspBev) : base(newEspBev) {
        Console.WriteLine("Adding Espresso Shot");
        Conso;e.WriteLine("Adding Steamed Milk");
    }
    string getDescription(){
        return tempEspresso.getDescription() + ", Extra Espresso Shot";
    }
    double getCost(){
        return tempEspresso.getCost() + 1.00;
    }

}